src/rules/and/marginals.jl

```diff
@@ -0,0 +1,8 @@
+@marginalrule AND(:in1_in2) (
+    m_out::Bernoulli,
+    m_in1::Bernoulli,
+    m_in2::Bernoulli
+) = begin
+    pin1, pin2, pout = mean(m_in1), mean(m_in2), mean(m_out)
+    return Contingency([(1-pin1)*(1-pin2)*(1-pout) (1-pin1)*pin2*(1-pout); pin1*(1-pin2)*(1-pout) pin1*pin2*pout])
```

Comment:
Furthermore, this `contingency` object has two dimensions and is asymmetric. As a user I would like to know how to interpret this table and therefore require something like axes labels. Perhaps an extension of the `Contingency` object is required for this purpose.

Reply:
Great suggestion!

Reply:
Any ideas on how to introduce axes for this structure?

Reply:
We could add these fields in the `Contingency` structure. However, I am not sure whether we can extract the variable names in the `rule` macro. I think we need to consult @bvdmitri for this.